recipes/sphinxjp.themecore/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.2.0" %}
+{% set sha256 = "55729fd3f70b0073da007829b54f533296a9787f03d68a45d747192fcae8c2ad" %}
+
+package:
+  name: sphinxjp.themecore
+  version: {{ version }}
+
+source:
+  fn: sphinxjp.themecore-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/s/sphinxjp.themecore/sphinxjp.themecore-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - docutils
+    - sphinx
```

Comment:
We should be able to drop `docutils` and `sphinx` from `build`. However, that doesn't need to be dealt with here.